indexd/app.py

```diff
@@ -11,9 +11,12 @@
 import sys
 import cdislogging
 
+import ddtrace
+
 
 def app_init(app, settings=None):
     app.logger.addHandler(cdislogging.get_stream_handler())
+    ddtrace.patch_all()
```

Comment:
Is this all that's required to inject data dog tracing into the existing log statements?

Reply:
https://ddtrace.readthedocs.io/en/stable/api.html#ddtrace.patch_all

> Automatically patches all available modules.